lib/generators/react_on_rails/base_generator.rb

```diff
@@ -62,7 +62,7 @@ def update_git_ignore
           /app/assets/javascripts/generated/*
         DATA
 
-        dest_file_exists?(".gitignore") ? append_to_file(".gitignore", data) : puts_setup_file_error(".gitignore", data)
+        dest_file_exists?(".gitignore") ? append_to_file(".gitignore", data) : options.errors.push(return_setup_file_error(".gitignore", data))
```

Comment:
@justin808 The problem is rails generators don't act like normal ruby classes. They get invoked, and each public method gets called from top to bottom. 

By using options, I can pass the array object from the install generator to any of the invoked generators and have them add errors.

Also, I couldn't find any information or examples on this issue.